app/views/directives/header/header.html

```diff
@@ -40,7 +40,7 @@
       <li role="menuitem"><a ng-href="project/{{currentProjectName}}/create?tab=deployImage">Deploy Image</a></li>
       <li ng-if-end role="menuitem"><a ng-href="project/{{currentProjectName}}/create?tab=fromFile">Import YAML / JSON</a></li>
 
-      <li ng-if-start="catalogLandingPageEnabled" role="menuitem"><a href="./">Browse Catalog</a></li>
+      <li ng-if-start="catalogLandingPageEnabled" role="menuitem"><a href="project/{{currentProjectName}}/catalog">Browse Catalog</a></li>
```

Comment:
Ideally we'd add a filter to build the URL using the `Navigate` service, similar to

https://github.com/openshift/origin-web-console/blob/master/app/scripts/filters/resources.js#L469